README.md

```diff
@@ -34,7 +34,7 @@ Another important piece of Chiquito are the signals. They represent elements of
 There are several types:
 
  + Internal signals: they are private for a particular step, and cannot be constraints for other steps.
- + Shared signals: they are shared by all steps in the circuit, and they can be constraints for their values on relative step instances (**rotation**). For example, if "a" is a shared signal, you could assert in a step type setup that `a == a.rot(2)` which means that `a` is equal to `a` in the next of the next step (super rotation `+2`).
+ + Shared signals: they are shared by all steps in the circuit, and they can be constraints for their values on relative step instances (**rotation**). For example, if "a" is a shared signal, you could assert in a step type setup that `a == a.rot(2)` which means that `a` is equal to `a` of the next step (super rotation `+2`).
```

Comment:
@aliatiia rot(2) actually mean the next of the next, while rot(1) is just the next. Perhaps there is a better way of explaining this

Reply:
Maybe emphasis with _italic_